src/go/rpk/pkg/config/manager.go

```diff
@@ -488,6 +484,7 @@ func v21_1_4MapToNamedSocketAddressSlice(
 // Redpanda version <= 21.4.1 only supported a single TLS config. This custom
 // decode function translates a single TLS config-equivalent
 // map[string]interface{} into a []ServerTLS.
+//nolint:revive // Using underscore here is intended to denote Redpanda version
```

Comment:
(also we want to remove this code down the line)

As a nit which is unimportant since we're removing this in the future: the `//nolint:revive` should come on the following line after the function signature so that it's not a part of the godoc